modules/BTBidAdapter.js

```diff
@@ -89,14 +104,25 @@ function bidResponse(buildBidResponse, bid, context) {
  * @returns {boolean} True if the bid request is valid, false otherwise.
  */
 function isBidRequestValid(bid) {
-  const { ab, siteId } = bid.params;
+  const { blockthrough } = bid.params;
+  if (!isPlainObject(blockthrough) || !Object.keys(blockthrough).length) {
+    logWarn(
+      'BT Bid Adapter: a object type "blockthrough" with site ids and adblock data must be provided.'
```

Comment:
Extremely nitpicky, but `an object type "blockthrough"` might sound a bit better

Reply:
also, maybe `site ids` should be changed. as we expect only 1 site id and 1 org id, right?